src/utils/buffer.ts

```diff
@@ -0,0 +1,8 @@
+export function toArrayBuffer(buf: Buffer) {
+    var ab = new ArrayBuffer(buf.length);
```

Comment:
Why use `var`? Also, there is a way to convert Buffer to ArrayBuffer without copying - https://github.com/jhiesey/to-arraybuffer/blob/master/index.js

Reply:
Copypast from stackoveflow (C)

Reply:
```
		var arrayCopy = new Uint8Array(buf.length)
		var len = buf.length
		for (var i = 0; i < len; i++) {
			arrayCopy[i] = buf[i]
		}
		return arrayCopy.buffer
```
I don't see too much difference between this two solutions, or virgilCrypto.encrypt/decrypt returns Uint8Array?

Reply:
Buffer is `instanceof Uint8Array` both in node.js and in the browser polyfill

Reply:
I don't have anything against copying and pasting code from StackOverflow, but please be mindful about that. At least make it adhere to the project's coding conventions.